src/java.base/share/classes/java/security/Security.java

```diff
@@ -259,7 +257,7 @@ private static ProviderProperty getProviderProperty(String key) {
             }
         }
 
-        return entry;
+        return null;
```

Comment:
If `entry` is always null, then we don't need to declare it at all.

Reply:
Agreed.